lib/routes/geocaching/blogs.ts

```diff
@@ -21,35 +35,82 @@ export const route: Route = {
         },
     ],
     name: 'Official Blogs',
-    maintainers: ['HankChow'],
+    maintainers: ['HankChow', 'Konano'],
     handler,
     url: 'geocaching.com/blog/',
 };
 
+const languageToCategory = { de: 140, fr: 138, es: 702, nl: 737, cs: 1404 };
+const languageToLabel = { de: 'Deutsch', fr: 'Français', es: 'Español', nl: 'Nederlands', cs: 'Čeština' };
+
 async function handler(ctx) {
     const baseUrl = 'https://www.geocaching.com';
-    const { data: response } = await got(`${baseUrl}/blog/wp-json/wp/v2/posts`, {
-        searchParams: {
-            per_page: ctx.req.query('limit') ?? 100,
```

Comment:
Why remove the ability to use [common parameter](https://docs.rsshub.app/guide/parameters#limit-entries) to change the page size?

Reply:
fixed in https://github.com/Konano/RSSHub/commit/d8372d07071a844fb52bbab2ee9f4afee26709e4

Reply:
fixed in https://github.com/DIYgod/RSSHub/commit/df7a5467f454f242c80288a285279f68b4d979e0